### PR TITLE
Adds interval-map-ref/bounds.

### PR DESCRIPTION
**Pull request by cfinegan**
This commit adds the `interval-map-ref/bounds` procedure to the `interval-map` library. `interval-map-ref/bounds` is like `interval-map-ref`, but it also returns the bounds of the interval associated with the queried position. If no mapping is found, then `#f` is returned for the start and end positions.

Example usage:
```
> (define im (make-interval-map '(((5 . 10) . "value"))))
> (interval-map-ref/bounds im 7)
5
10
"value"
> (interval-map-ref/bounds im 10 'not-found)
#f
#f
'not-found
```

### CODE REVIEW

File: data-doc/data/scribblings/interval-map.scrbl [hunk 1]
```diff
@@ -78,6 +78,17 @@ Return the value associated with @racket[position] in
 applied if it is a procedure, or returned otherwise.
 }
 
+@defproc[(interval-map-ref/bounds [interval-map interval-map?]
```
Comment (samth) [inline data-doc/data/scribblings/interval-map.scrbl]: This should probably have an `@history` annotation.

File: data-lib/data/interval-map.rkt [hunk 2]
```diff
@@ -31,6 +32,24 @@
                  (not-found)))]
           [else (not-found)])))
 
+(define (interval-map-ref/bounds im key [default not-given])
+  (define (not-found)
+    (cond [(eq? default not-given)
+           (error 'inverval-map-ref/bounds "no mapping found\n  key: ~e" key)]
```
Comment (samth) [inline data-lib/data/interval-map.rkt]: Spelling error here.

File: data-doc/data/scribblings/interval-map.scrbl (hunk 1 above)
Comment (cfinegan) [reply data-doc/data/scribblings/interval-map.scrbl]: I added the `@history` annotation and added a `version` field to data-lib's `info.rkt`, which I set to 1.1.

**Comment by jeapostrophe** [comment]
The code and docs look good to me

**Pull request merged**